tests/test_features_feature_collection.py

```diff
@@ -410,6 +388,49 @@ def quantiles_abs_diff(
     assert (res_df[expected_output_names[0]] != res_df[expected_output_names[2]]).any()
 
 
+def test_series_funcs(dummy_data):
+    def min_max_time_diff(x: pd.Series, mult=1):
+        diff = (x.index[1:] - x.index[:-1]).total_seconds().max()
```

Comment:
ben niet zo goed mee wat deze code doet -> hier doe je `max()`, dus dan heb je een scalar
-> op de volgende lijn doe je nogmaals `min()` & `max()`

Reply:
heb in de test code die eerste `max` in commentaar gezet & alles lijkt nog te werken 😃 
```python
        diff = x.index.to_series().diff().dt.total_seconds()#.max()
```